### DISCUSSION TITLE
ETA for release including 3.13.0?

### DISCUSSION THREAD

**Discussion by ottumm** [pyenv/pyenv#3088] [General] [answered]
Hello! I have two release-related questions:

1. When can we expect a new release that includes Python 3.13.0?
2. Is there a reason that pyenv 2.4.13 is tagged as the "latest" release in git, even though 2.4.14 exists?

Thank you!

**Comment by native-api** [comment, marked as answer]
> * When can we expect a new release that includes Python 3.13.0?

[Due to the new no-GIL flavor, there's an outstanding task before 3.13.0 can go live](https://github.com/pyenv/pyenv/pull/3081#issuecomment-2401126075).
I've allocated time for it and should be done withing 24 hours.

> * Is there a reason that pyenv 2.4.13 is tagged as the "latest" release in git, even though 2.4.14 exists?

Fixed. I thought I checked "set as the latest release" when doing 2.4.14. Might be a Github malfunction since I made the release in a bit nonstandard way to save time. Will pay attention to this on the next release to see if there is a malfuction.

**Reply by ottumm** [reply]
Great, thanks for the reply, that all works for me!

**Reply by native-api** [reply]
https://github.com/pyenv/pyenv/releases/tag/v2.4.15

**Comment by native-api** [comment]
If there's an urgent need to use 3.13.0, you can check out the `master` branch tip.

**Reply by wyatt-wong** [reply]
Well people install pyenv to manage Python versions but now it seems to other way round that it forbid user to check and update to the latest Python versions.

**Comment by manusa3190** [comment]
I just checked pyenv and it looks like Python 3.13 can now be installed!

```
% brew update
・・・
% brew upgrade pyenv
・・・
% pyenv install --list | grep 3.13
  3.13.0
  3.13.0t
  3.13-dev
  3.13t-dev
・・・
```

To whoever updated pyenv, thank you!!

**Comment by ottumm** [comment]
pyenv 2.4.15 has been released, and so Python 3.13.0 can be installed. Closing this discussion now. Thanks @native-api!
